apps/wallet/src/background/keyring/DerivedAccount.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    normalizeSuiAddress,
+    type Keypair,
+    type SuiAddress,
+} from '@mysten/sui.js';
+
+import { type Account, AccountType } from './Account';
+import { AccountKeypair } from './AccountKeypair';
+
+export type SerializedDerivedAccount = {
```

Comment:
Might at some point be worth making schemas for this so that we have some easily validation on deserialization as well, but can punt for now.